podvm/README.md

```diff
@@ -0,0 +1,134 @@
+# Introduction
+
+This directory contains the sources to build the podvm image (qcow2 file) for various Linux distributions and cloud providers. So use
+the instructions on the next sections if you need to build your own image with changes to meet your requirements. Otherwise you can
+find [here](../docs/consuming-prebuilt-podvm-images.md) information on how to consume pre-built images.
+
+# How to build locally
+
+In order to build locally it requires the source trees and softwares mentioned on the [developer's guide](../docs/DEVELOPMENT.md) to build this project binaries. It will also need [packer](https://www.packer.io/) (to build the qcow2), [rust](https://www.rust-lang.org/tools/install) (to build the Kata Containers's agent), as well as the following packages:
+
+* On Ubuntu:
+
+  ```$ apt-get install -y qemu-kvm cloud-utils qemu-utils protobuf-compiler pkg-config libdevmapper-dev libgpgme-dev```
+
+You may need to link the agent with the musl C library. In this case, you should install the musl-tools (Ubuntu) package and setup the Rust toolchain as explained [here](https://github.com/kata-containers/kata-containers/blob/CCv0/src/agent/README.md#build-with-musl).
+
+Finally run the following commands to build the qcow2 image:
+
+```
+$ export CLOUD_PROVIDER=[aws|azure|ibmcloud|libvirt]
+$ make image
+```
+
+# How to build within container
+
+This directory contains dockerfiles to build the podvm image entirely within container so that it only requires docker or podman installed on the host.
+
+In general it is needed to follow these steps:
+
+1. Build a builder container image for a given Linux distribution
+1. Build the podvm container image for a given cloud provider
+1. Extract the podvm image (qcow2 file) from the container image
+
+The next sections describe that process in details. Note that although the following examples use docker, it can be carried out with podman too.
+
+## Building a builder image
+
+The builder image packages the cloud-api-adaptor and Kata Containers sources as well as the softwares to build
+the binaries (e.g. *kata-agent* and *agent-protocol-forwarder*) that should be installed in the podvm image.
+
+The builder image is agnostic to cloud providers in the sense that one can be used to build for multiple providers, however it is
+dependent of the Linux distribution the image is built for. Therefore, in this directory you will find dockerfiles to each supported distributions, which are currently Ubuntu 20.04 ([Dockerfile.podvm_builder](./Dockerfile.podvm_builder)), CentOS Stream 8 ([Dockerfile.podvm_builder.centos](./Dockerfile.podvm_builder.centos)), and RHEL 8.7 ([Dockerfile.podvm_builder.rhel](./Dockerfile.podvm_builder.rhel)).
```

Comment:
nit: "of the Linux" => "on the Linux"
nit: "in this directory you will find dockerfiles to each supported distributions" => "in this directory you will find dockerfiles **for** each supported distributions"